tools/compile_tool/main.cpp

```diff
@@ -651,24 +610,7 @@ static std::map<std::string, std::string> parseConfigFile(char comment = '#') {
 }
 
 static std::map<std::string, std::string> configure() {
-    const bool isMYRIAD = FLAGS_d.find("MYRIAD") != std::string::npos;
-    auto config = parseConfigFile();
-
-    if (isMYRIAD) {
-        if (!FLAGS_VPU_NUMBER_OF_SHAVES.empty()) {
-            config[InferenceEngine::MYRIAD_NUMBER_OF_SHAVES] = FLAGS_VPU_NUMBER_OF_SHAVES;
-        }
-
-        if (!FLAGS_VPU_NUMBER_OF_CMX_SLICES.empty()) {
-            config[InferenceEngine::MYRIAD_NUMBER_OF_CMX_SLICES] = FLAGS_VPU_NUMBER_OF_CMX_SLICES;
-        }
-
-        if (!FLAGS_VPU_TILING_CMX_LIMIT_KB.empty()) {
-            config[InferenceEngine::MYRIAD_TILING_CMX_LIMIT_KB] = FLAGS_VPU_TILING_CMX_LIMIT_KB;
-        }
-    }
-
-    return config;
+    return parseConfigFile();
```

Comment:
There is only this line left in this `configure()` function. I suggest replacing `configure()` calls with `parseConfigFile()`.

Reply:
fixed